packages/cli/openapi-parser/src/v3/converters/convertPathItem.ts

```diff
@@ -409,20 +471,43 @@ function convertToEndpoint({
             document,
             context,
             requestBreadcrumbs: [...requestBreadcrumbs, "Body"],
+            isStreaming,
+            streamCondition,
         });
     } else if (operation.requestBody != null) {
         convertedRequest = convertRequest({
             requestBody: operation.requestBody,
             document,
             context,
             requestBreadcrumbs: [...requestBreadcrumbs],
+            isStreaming,
+            streamCondition,
         });
     }
 
+    // TODO: We need to convert the operation.responses into a structure with the same code, content-type, etc.
+    let responses = operation.responses;
+    if (streamCondition != null) {
+        const ref = isStreaming ? streamCondition.responseStream : streamCondition.response;
+        responses = {
+            "200": {
+                description: "",
+                content: {
+                    "application/json": {
+                        schema: {
+                            $ref: ref,
+                        },
+                    },
+                },
+            },
+        };
+    }
```

Comment:
Need to fix this - it's hard-coded as-is. The `code` and `content-type` values are variable strings, so we'll need to map those to the new `operation.responses` value, but it isn't sufficient to just reference the `code` and `cotentType` properties.